packages/commons/src/auth/authorization.ts

```diff
@@ -0,0 +1,53 @@
+import { unauthorizedError } from "pagopa-interop-models";
+import { AppContext, AuthData, UserRole } from "../index.js";
+
+export function hasUserRole(
+  authData: AuthData,
+  userRoles: UserRole[]
+): boolean {
+  return (
+    authData.tokenType === "ui" &&
+    authData.userRoles.some((role: UserRole) => userRoles.includes(role))
+  );
+}
+
+export function validateAuthorization<
```

Comment:
Can we add a jsdoc that explains what token types are, etc...?

Reply:
Done!